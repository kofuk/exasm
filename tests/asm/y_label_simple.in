@L1 nop
j @L1
nop
