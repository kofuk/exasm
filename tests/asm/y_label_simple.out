@00 00000000 00000000 // nop
@02 11000000 11111100 // j -0x04
@04 00000000 00000000 // nop
